.github/workflows/tag.yml

```diff
@@ -123,22 +123,44 @@ jobs:
         run: |
           echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\/v/}
           echo ::set-output name=TAG::${GITHUB_REF/refs\/tags\//}
+
+      - name: Checkout Homebrew-tap
+        uses: actions/checkout@master
+        with:
+          repository: GitGuardian/homebrew-tap
+          token: ${{ secrets.PAT_GITHUB }}
+          path: ./tap
+
       - name: Checkout Homebrew-ggshield
         uses: actions/checkout@master
         with:
           repository: GitGuardian/homebrew-ggshield
           token: ${{ secrets.PAT_GITHUB }}
-          path: ./brew
+          path: ./ggshield
```

Comment:
I am surprised, don't you want to checkout it in the brew directory like it was done before?

Reply:
We checkout this repo to `./ggshield` now.

Reply:
there is already a ggshield directory in the repo root, don't you want to use another non existent one?

Reply:
Should be good now.